### PR TITLE
pass in explicit local_java_runtime target name instead of parsing it…

### PR DESCRIPTION
**Pull request by Wyverald**
… from the canonical repo name

### CODE REVIEW

**Comment by Wyverald** [comment]
cc @fmeum

**Review by comius — DISMISSED** [review]

File: toolchains/local_java_repository.bzl [hunk 1]
```diff
@@ -202,7 +197,7 @@ local_java_runtime(
     java_home = "%s",
     version = "%s",
 )
-""" % (repository_ctx.name, runtime_name, java_home, version)
```
Comment (comius) [inline toolchains/local_java_repository.bzl:205]: Just one question out of curiosity. repository_ctx.attr.name is the same as repository_ctx.name?

Comment (Wyverald) [reply toolchains/local_java_repository.bzl:205]: yep

**Comment by comius** [comment]
Buildifier is complaining:

<a href="https://github.com/bazelbuild/rules_java/blob/9b5ea1128461779f74ed3306e81d41872ff74bd4/toolchains/local_java_repository.bzl#L242">toolchains/local_java_repository.bzl:242:9</a>: <a href="https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#unsorted-dict-items">unsorted-dict-items</a>: Dictionary items are out of their lexicographical order.</pre></code>

**Comment by Wyverald** [comment]
> Buildifier is complaining:
> 
> [toolchains/local_java_repository.bzl:242:9](https://github.com/bazelbuild/rules_java/blob/9b5ea1128461779f74ed3306e81d41872ff74bd4/toolchains/local_java_repository.bzl#L242): [unsorted-dict-items](https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#unsorted-dict-items): Dictionary items are out of their lexicographical order.

Fixed.

**Comment by Wyverald** [comment]
hmm, apparently the commit has already been merged. Do I need to close this PR and send a new one to fix the buildifier error?

**Comment by comius** [comment]
> hmm, apparently the commit has already been merged. Do I need to close this PR and send a new one to fix the buildifier error?

Wait what? How?

Copybara should open a cl and then close the pr.

**Comment by comius** [comment]
> > hmm, apparently the commit has already been merged. Do I need to close this PR and send a new one to fix the buildifier error?
> 
> Wait what? How?
> 
> Copybara should open a cl and then close the pr.

But since you lgtm-ed previous version, we probably need a new pr to appease copybara :)

**Comment by fmeum** [comment]
Could this be included in a 5.0.1 release for the BCR?

**Pull request closed without merging**